### ISSUE TITLE
After recent Chrome Update (115.0.5790.170), Extension No Longer Works

### ISSUE THREAD

**Issue by jdtechsol** [ynshung/better-yt-shorts#67]
Error as seen in devtools console:
content-script.js:373 Uncaught TypeError: Cannot read properties of undefined (reading 'children')
    at content-script.js:373:57
Chrome Version  115.0.5790.170

**Comment by allojys** [comment]
So do I.😓

**Comment by DJ5D** [comment]
Same here. It no longer works as of 8-4-2023

**Comment by pioliX000** [comment]
Same on Firefox, everything is broken except for going to the next and previous short, and rewinding and forwarding with arrow keys. Otherwise everything else is just broken, mainly the volume slider and the time slider,

**Comment by pioliX000** [comment]
![image](https://github.com/ynshung/better-yt-shorts/assets/65280877/90ee6a82-9f06-48cb-a8cd-2c3d9ac4afc5)
Same error as well

**Comment by emyrdemir** [comment]
#69

**Comment by ynshung** [comment]
It seems like an internal update on the YouTube website that broke the extension, will publish the update soon and hopefully Chrome will update on the store asap. For the temporary fix you can go to @emyrdemir's fork and load the extension (unpacked) from there.

**Comment by ynshung** [comment]
Firefox's update is now live! Waiting for Google... (which always take a long time)

**Comment by allojys** [comment]
> Firefox's update is now live! Waiting for Google... (which always take a long time)

firefox extension is updated but still not working. Still not changed

```javascript
  var progBarList = overlayList.children[2].children[0].children[0];
```
this should be changed to

```javascript
  var progBarList = overlayList.children[3].children[0].children[0];

```

**Comment by ynshung** [comment]
Huh weird, I just reupload the extension again, hopefully it'll work now.

**Comment by allojys** [comment]
It's now working thank you😊

**Comment by ynshung** [comment]
Attention for **Chrome** user: There are some issue when I was trying to update and the package and currently the extension got unpublished. I will try to republish it as soon as possible once Google is done reviewing the package

Sidenote: Updating extension in Chrome Web Store is a nightmare

**Comment by Maouniel** [comment]
> you can download the latest version here and load the extension unpacked

And now for some reason shorts automaticaly scrolling down every second if not paused.

**Comment by ynshung** [comment]
> > you can download the latest version here and load the extension unpacked
> 
> And now for some reason shorts automaticaly scrolling down every second if not paused.

Have you tried reinstalling the extension and made sure the auto scroll below like count is disabled?

**Comment by ynshung** [comment]
> Attention for **Chrome** user: There are some issue when I was trying to update and the package and currently the extension got unpublished. I will try to republish it as soon as possible once Google is done reviewing the package
> 
> Sidenote: Updating extension in Chrome Web Store is a nightmare not only [suffered](https://support.google.com/chrome/thread/147692064/urgent-accidentally-unpublished-chrome-extension?hl=en) [by me](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/fhksXFZ6K_c/m/idz07bBkAAAJ?pli=1). With the current trademark issue with the logo design, I'm not sure whether the review will be approved anytime soon. Worse-case scenario is that we have to wait until the logo is redesigned for publication and the past ratings/users are not preserved. 

Update: The store listing is up now! Currently pending review for update to 2.7.2

**Comment by ynshung** [comment]
The chrome web store is now updated, can anyone confirm that it is now working? Make sure the version is `2.7.2`.

**Comment by dexman545** [comment]
The progressbar shows timestamps, but trying to jump to a time is broken (got to random points or the begining)

**Comment by Maouniel** [comment]
> > > you can download the latest version here and load the extension unpacked
> > 
> > 
> > And now for some reason shorts automaticaly scrolling down every second if not paused.
> 
> Have you tried reinstalling the extension and made sure the auto scroll below like count is disabled?

Just reinstalled from store. 2.7.2, autoskip disabled. 
Problem is still here. Console says: Skipping short that had 0 likes.

**Comment by ynshung** [comment]
Currently it seems like the extension is fully up and running, for the other issue @dexman545 @Maouniel could you open a separate issue? I had tried reproducing it but failed to do so. Thanks!

Another potential fix is to clear local storage by navigating to a YouTube page and opening Developer tools > Application tab > Local Storage > https://youtube.com > Clear all icon
